### DISCUSSION TITLE
My request as a teacher was approved on Nov 16. But I can not use Copilot now. What should I do the next?

### DISCUSSION THREAD

**Discussion by sunstudio** [community/community#144887] [GitHub Education] [answered]
### Hi everyone,

My request as a teacher was approved on Nov 16. But I can not use Copilot now. What should I do the next?
![image](https://github.com/user-attachments/assets/879de1a0-87da-4e26-b3b8-8980ddbc96a1)

**Comment by Gesronet** [comment, marked as answer]
If your request for GitHub Copilot as a teacher was approved on November 16 but you're still unable to access it, here are some steps you can take:

**1). Check Your Email:** Ensure you received a confirmation email regarding your approval. Sometimes, additional steps or links are provided.

**2). Log Out and Log In:** Try logging out of your GitHub account and then logging back in. This can refresh your permissions.

**3). Clear Browser Cache:** Clear your browser's cache and cookies, then try reaccessing Copilot.

**4). Verify Subscription:** Go to your GitHub settings and check if the Copilot subscription is active under "Billing & plans."

**5). Contact Support:** If none of the above steps work, consider reaching out to GitHub Support for assistance. Provide them with details about your approval and the issues you're facing.

Hopefully, one of these steps will help you gain access to Copilot. Goodluck!!!

**Reply by parcox** [reply]
Thank you. This solved my problem, especially step four. It should be activated first before doing anything else.
